reference/fleet/example-kubernetes-fleet-managed-agent-helm.md

```diff
@@ -3,7 +3,7 @@ mapped_pages:
   - https://www.elastic.co/guide/en/fleet/current/example-kubernetes-fleet-managed-agent-helm.html
 applies_to:
   stack: ga 8.18
```

Comment:
Not a line you changed, but indicating that this feature is supported only on `8.18` doesn't seem right. I posted a question in the slack channel.